akka-http-core/src/test/scala/akka/http/impl/util/ExampleHttpContexts.scala

```diff
@@ -43,7 +43,7 @@ object ExampleHttpContexts {
     context.init(null, certManagerFactory.getTrustManagers, new SecureRandom)
 
     val params = new SSLParameters()
-    Java6Compat.setEndpointIdentificationAlgorithm(params, "https")
+    Java6Compat.trySetEndpointIdentificationAlgorithm(params, "https") // TODO, well this is akka private
```

Comment:
ignore the TODO (marked only as such because we did point people to this file, and it may have confused them why they can't call this)